_posts/2021-12-16-log4j-patch-releases.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title:  "Apache Flink Log4j emergency releases"
+date:   2021-12-16 00:00:00
+categories: news
+authors:
+- chesnay:
+  name: "Chesnay Schepler"
+
+---
+
+The Apache Flink community has released emergency bugfix versions of Apache Flink for the 1.11, 1.12, 1.13 and 1.14 series.
+
+These releases include a version upgrade for Log4j to address [CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228).
```

Comment:
I think it worth a mention of CVE-2021-45046, wdyt?

Reply:
yes, that's a good idea.